### DISCUSSION TITLE
Interface DJango Admin ?

### DISCUSSION THREAD

**Discussion by williank** [openwisp/docker-openwisp#339] [Q&A]
I have openwisp installed, but when I access my dashboard.*** what came up was a django admin panel. How do I access the intuitive interface?

**Comment by nemesifier** [comment]
@williank, can you please share:

- A screenshot of what you're seeing
- A screenshot of what you're expecting (are you looking at something specific from the docs?)

**Comment by nemesifier** [comment]
Closing for lack of follow up.
